tls/x509utils/certpool/utils.go

```diff
@@ -0,0 +1,93 @@
+package certpool
```

Comment:
why is this not in `darvaza.org/core`?

Reply:
maturity mostly, but you are right. it should end `core`

Reply:
cleaned up in preparation for `darvaza.org/core.List[T]`